templates/macros/community.html

```
@@ -0,0 +1,40 @@
+    {%- elif ratio < 2.6 -%}34
+    {%- elif ratio < 4.2 -%}28
+    {%- elif ratio < 5.5 -%}25
+    {%- else -%}22
+    {%- endif -%}
+  {%- else -%}24
+  {%- endif -%}
+{% endmacro logo_height %}
+
+{# One full set of logos. The marquee renders it twice; the copy is hidden from assistive tech. #}
+{% macro logo_set(participants, duplicate=false) %}
+<div class="community-logo-set"{% if duplicate %} aria-hidden="true"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)